### DISCUSSION TITLE
What would it take to make a Radix + Tailwind version?

### DISCUSSION THREAD

**Discussion by triptu** [KevinVandy/material-react-table#1123] [General]
Hi! Thanks for creating this package. Going through the [roadmap](https://www.material-react-table.com/roadmap), I believe there are some refactoring efforts planned to allow for Joy and Base UI versions. I was planning to fork and remove Material UI dependency from the package, substituting it for Radix and Tailwind. What are some things I should be aware of to make this easier? And would it be better to do this with [mantine-react-table](https://github.com/KevinVandy/mantine-react-table) instead?

**Comment by anindosarker** [comment]
Check out [here](https://ui.shadcn.com/docs/components/data-table)

**Reply by triptu** [reply]
Thanks, I have seen it. It's good as a general overview of how to go about building a data grid, but would required a lot of ground up work to get close to MRT.

**Reply by anindosarker** [reply]
Yes. That is why we're still stuck on material ui because that table is very much important in many projects
